scene/main/viewport.cpp

```diff
@@ -1985,13 +1985,16 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 
 		if (gui.drag_data.get_type() == Variant::NIL && over && !gui.modal_stack.empty()) {
 
+			//  Dragging nodes causes re-search of currently moused-over control
+			Control *dragged = _gui_find_control(mpos);
+
 			Control *top = gui.modal_stack.back()->get();
 
-			if (over != top && !top->is_a_parent_of(over)) {
+			if (dragged != top && !top->is_a_parent_of(dragged)) {
```

Comment:
This no longer uses nor updates `over`, which makes me think that it's likely going to break some functionality.

Reply:
When 'over' was updated - it worked as well, but introduced one bug with dragging sliders (when I opened color picker, and dragger over one of RGB sliders, and released button, moved cursor somewhere, and then returned to that slider - it was still draging, even I didn't hold any buttons). With change from above that issue was gone. As you said - there is no modification for 'over', thus whole code block is responsible for one very specific case (sliding over menu items with Popups expanded), so it shouldn't affect any other functionalities when mouse is moving.

Reply:
In fact, in https://github.com/godotengine/godot/pull/32943 - 'over' is being updated when that 'drag' event is being detected.